### PR TITLE
Feature backend model initialization

### PR DESCRIPTION
**Pull request by IanMcC123**
This is the first iteration of our database models for item posts and users. This code is subject to change during the evolution of our project, as we anticipate there will be minor changes throughout. Prior to merging, this code must be commented and reviewed by multiple team members.

### CODE REVIEW

**Review by shihab3750 — APPROVED** [review]
Looks good.

File: backend/models/message.model.js
Comment (AmineSlaoui) [inline backend/models/message.model.js:1]: What is the difference between conversation and content in the file message.model.js

Comment (George-Hachem) [reply backend/models/message.model.js:1]: content is the message itself and the conversation attribute is there to connect every message to the conversation

Comment (shihab3750) [reply backend/models/message.model.js:1]: It is a list of conversations. So each conversation also has content. They are ordered based on time.

**Review by AmineSlaoui — APPROVED** [review]
Looks good

**Pull request merged**